app/controllers/finance/financial_links_controller.rb

```diff
@@ -36,7 +36,7 @@ def show
   end
 
   def create
-    @financial_link = FinancialLink.new
+    @financial_link = FinancialLink.first_unused_or_create
```

Comment:
It looks as if this would better be put in a transaction (creating the link and adding the bank transaction).

Reply:
why?
1) `first_unused_or_create` is a generic function, needed at different places. 
2) i see no problem happeing when adding the the bank transaction fails

Reply:
1. To be clear: I mean in the controller, not in the method.
2. That's true! So we can leave it :+1: